The sample binary will fail to link on systems that need a replacement for
calloc because their implementation isn't GNU-calloc compatible. For example,
OS X is affected by this.

Signed-off-by: Clemens Lang <cal@macports.org>
Upstream-Status: Submitted [http://lists.gnu.org/archive/html/bug-barcode/2015-10/msg00000.html]
--- Makefile.am.orig	2015-10-19 19:14:29.000000000 +0200
+++ Makefile.am	2015-10-19 19:14:36.000000000 +0200
@@ -38,4 +38,4 @@
 
 
 sample_SOURCES = sample.c
-sample_LDADD = libbarcode.la
+sample_LDADD = libbarcode.la lib/libgnu.la
